### PR TITLE
Add documentation about how to run lint and fix dependencies

### PR DESCRIPTION
**Pull request by andybalaam**
---
This change is marked as an *internal change* (Task), so will not be included in the changelog.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -179,6 +179,31 @@ Ensure you've followed the above development instructions and then:
 yarn test
 ```
 
+Running lint
```
Comment (turt2live) [inline README.md:182]: should these sections be in the contributing guide instead?

also we're mixing title styles: please let's not do that :p

Comment (andybalaam) [reply README.md:182]: > also we're mixing title styles: please let's not do that :p

Fixed in https://github.com/matrix-org/matrix-react-sdk/pull/8705

Comment (andybalaam) [reply README.md]: Apologies for the force push, I rebased on top of #8705

**Comment by andybalaam** [comment]
I am away next week, so please merge if you tick this while I am away.

**Review by turt2live — APPROVED** [review]
thanks :)

**Pull request merged**